Makefile

```diff
@@ -493,17 +494,24 @@ merge-to-master-release:
 	$(Q)docker build -f Dockerfile.rhel -t $(OPERATOR_IMAGE_REF) .
 	docker push "$(OPERATOR_IMAGE_REF)"
 
+.PHONY: verify-image-using-sha256-digest
+## The image pushed to registry should be verified
+verify-image-using-sha256-digest:
+	$(eval AIR_GAP_OPERATOR_IMAGE_REF := $(shell docker inspect --format="{{index .RepoDigests 0}}" $(OPERATOR_IMAGE_REF)))
+	docker rmi "$(AIR_GAP_OPERATOR_IMAGE_REF)"
+	docker rmi "$(OPERATOR_IMAGE_REF)"
```

Comment:
these two lines could be combined into single one:

```
docker rmi image1 image2
```

Reply:
Done. a4617d0de4c0f7b730b1054e06d9d384d8a4be11